### PR TITLE
docs: Improved README.md and CONTRIBUTION.md, Fixed typos and improved sentences

### PR DESCRIPTION
**Pull request by JoeriSmits**
### Please check if the PR fulfills these requirements ###

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Docs update for README.md and CONTRIBUTION.md. It fixes some typos and the quality of some sentences.

**What is the current behavior? (You can also link to an open issue here)**
Has some typos. The doc can be improved.

**What is the new behavior (if this is a feature change)?**
Docs improved

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**
-

**Other information:**
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,18 +120,18 @@ yarn test:coverage
...
 ```
 
-Important thing now is all those packages are linked together so changing something in `@deity/falcon-client` will be automatically reflected in `shop-with-blog` example
+Important thing now is all those packages are linked together so changing something in `@deity/falcon-client` is automatically reflected in the `shop-with-blog` example.
 
 ### TODO
 - jest for all packages
 - changelog generation? - https://github.com/lerna/lerna-changelog
-- learna publish packages to npm script?
+- learning publish packages to npm script?
```
Comment (Borales) [inline README.md]: @JoeriSmits `lerna` is the name of the "monopero" management tool, it's not a typo :)

Comment (JoeriSmits) [reply README.md]: @Borales Check :). I did not know that. I will update the PR.

**Comment by Borales** [comment]
@JoeriSmits actually, just noticed that Codacy fails with a few minor issues, could you please take a look at it? https://app.codacy.com/app/Borales/falcon/pullRequest?prid=2305032

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -71,10 +70,10 @@ Based on https://conventionalcommits.org/
 - `build:` - a commit that with changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
 - `ci:` - a commit that with changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
 - `docs:` - a commit that with changes documentation only changes
-- `perf:` - a commit with changes that improves performance
-- `refactor:` - a commit with changes that neither are fixes a bug nor adds a feature
-- `style:` - a commit with that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-- `test:` - a commit with adding missing tests or correcting existing tests
+- `perf:` - a commit with changes that improve performance
```
Comment (cane-lagerwaard) [inline .github/CONTRIBUTING.md]: "improves" is correct

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -71,10 +70,10 @@ Based on https://conventionalcommits.org/
...
+- `refactor:` - a commit with changes that neither are fixes, a bug or adds a feature
```
Comment (cane-lagerwaard) [inline .github/CONTRIBUTING.md]: please change back to previous sentence, it was correct

Comment (cane-lagerwaard) [reply .github/CONTRIBUTING.md]: referring to: "a commit with changes that neither are fixes a bug nor adds a feature"

**Pull request merged**